docs/dev-container.md

```diff
@@ -0,0 +1,61 @@
+# Developing inside a Container
+
+## Overview
+
+This document describes how to configure and develop in the dev container
+specified by this repository. The goal is to provide developers with the same
+development environment that is mainly isolated from their host. This method
+promotes reproducibility and remove the need to install dependencies on the host
+such as Node.js or Python. For additional information on VS Code development
+containers, see [Developing inside a Container].
+
+## Requirements
+
+- [Docker]
+- [VS Code] and the extension [Remote - Containers]
+
+## Quick Start
+
+Open the workspace folder in VS Code after cloning it. VS Code should invite you
+to open the folder in the dev container specified by this repository.
+Alternatively:
+
+- Press `F1` or click on `Help` > `Show All Commands`
```

Comment:
I don't have "F1" on my external keyboard 😄. We could also add shortcuts for vscode to open "All Commands". But it is not necessary change and you can make it call.

<kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (macOS) or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (Windows)

```suggestion
- Press `F1` or <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (macOS) or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> (Windows) or click on `Help` > `Show All Commands`
```